packages/react-sdk/src/components/Whiteboard/SvgCanvas/SvgCanvas.tsx

```diff
@@ -176,9 +242,12 @@ export const SvgCanvas = forwardRef(function SvgCanvas(
                   }),
             }}
             viewBox={`0 0 ${viewportWidth} ${viewportHeight}`}
+            onClick={handleCanvasClick}
             onMouseDown={onMouseDown}
             onMouseMove={handleMouseMove}
-            onWheel={onWheel}
+            onKeyDown={handleKeyDown}
+            onKeyUp={handleKeyUp}
+            tabIndex={0}
```

Comment:
I suggest to apply these changed to infinite canvas mode only. 

For both modes it also adds a focus default outline. On non-infinite it can be seen directly. On infinite user needs to scroll to the edge. I think for infinite we should hide the effect.

I also think we should use `tabIndex={-1}` for inf mode. Having `0` adds extra focus to the whole canvas when user switches from top to bottom bars and this focus is not visible because the canvas is infinite.